src/Features/VisualBasic/Portable/UseAutoProperty/VisualBasicUseAutoPropertyCodeFixProvider.vb

```diff
@@ -63,19 +63,32 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.UseAutoProperty
                 statement = statement.WithAsClause(initializer.asNewClause)
             End If
 
+            If initializer.arrayBounds IsNot Nothing Then
+                Dim semanticsModel = Await propertyDocument.GetSemanticModelAsync(cancellationToken).ConfigureAwait(False)
+                Dim arrayType = DirectCast(semanticsModel.GetDeclaredSymbol(propertyDeclaration, cancellationToken).Type, IArrayTypeSymbol)
```

Comment:
i don't know why this cast is necessarily safe.  can you make a try/cast?

Reply:
Seeing `ArrayBounds` should means it's an array type. But it's OK to be defensive and robust in analyzers layer.